docs/analysis-flow-scripts/register_example_file.py

```diff
@@ -14,11 +14,13 @@
     var_index=bt.Gene.ensembl_gene_id,
     categoricals={
         "cell_type": bt.CellType.name,
+        "cell_type_id": bt.CellType.ontology_id,
```

Comment:
Why was this needed? 🤔

Reply:
```
• 1 non-validated categories are not saved in Feature.name: ['cell_type_id']!
      → to lookup categories, use lookup().columns
      → to save, run add_new_from_columns
```

is what you get otherwise

Reply:
Ok 🤔 

Is that a warning or does it prevent `.validate()` from succeeding? What if I'm not interested in validating that column? Often times, I'd like to keep a few "raw columns" around and only want to validate the "gold columns", right?